code/baselines/wsdbaselines_track1_table.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+
+import pandas as pd
+import numpy as np
+import sys
+from pathlib import Path
+
+
+def main():
+    dfs = []
+    for p in Path('./').glob('*.scores.tsv'):
+        mdf = pd.read_csv(p, sep=': +', header=None, engine='python')
+        mdf.rename(columns={0: 'metric', 1: 'value'}, inplace=True)
+        ff = p.name.split('.')
+        mdf['method'] = ff[0]
+        mdf['lang'] = ff[1]
+        mdf['part'] = ff[2]
+        dfs.append(mdf)
+
+    mdf = pd.concat(dfs, ignore_index=True)
+    results = mdf.groupby(['method', 'lang', 'part', 'metric']).value.agg(lambda r:
```

Comment:
Indentation formatting issue.

Reply:
What is the issue? I don't see any issues, and my Pycharm doesn't as well.

Reply:
This line is very long and github may format it ugly, depending on your screen width and browser.
![Screenshot from 2024-08-06 22-48-29](https://github.com/user-attachments/assets/6ff54bf9-0cb0-457f-a375-1d911586e89b)
 So one must look very attentively to see that ({len... is not on a new line)  . One possible solution is formatting like 
`.agg(
        lambda r:
        f'{np.mean(r):.3f}+-{np.std(r):.3f} ({len(r)})'
    ).reset_index()` then it will fit even into the recommended 79 characters width)

Reply:
fixed